WordPress/Classes/ViewRelated/Post/PostSettings/Views/PostSettingsCategoriesRow.swift

```diff
@@ -0,0 +1,45 @@
+import SwiftUI
+import WordPressUI
+import WordPressData
+
+struct PostSettingsCategoriesRow: View {
+    let categories: [String]
+
+    var body: some View {
+        HStack {
+            PostSettingsIconView("wpdl-category")
```

Comment:
Use the `square.grid.2x2` SF symbol?

Reply:
These are new symbols from WPDS, which I used consistently on all new and existing screens with a few exceptions. The main one is the context menus. Makes it look and feel more WordPress.

We need to find a way to include all the new icons at some point.